argus/tests/cloud/smoke.py

```diff
@@ -75,6 +80,91 @@ def test_password_set(self):
         self.assertEqual('1', stdout.strip())
 
 
+class TestUpdatePasswordSmoke(base.TestBaseArgus):
+
+    def _run_remote_command(self, password, cmd):
+        remote_client = self.manager.get_remote_client(
+            CONF.cloudbaseinit.created_user, password)
+        stdout = remote_client.run_command_verbose(cmd)
+        return stdout
+
+    def _random_password(self, length=10):
+        random_generator = random.SystemRandom()
+        alphabet = string.ascii_letters + string.digits
+        return ''.join(random_generator.choice(alphabet)
+                       for _ in range(length))
+
+    def _update_password(self, password):
+        url = "http://%(host)s:%(port)s/password" % {
+            "host": self.manager.services[1].host,
+            "port": self.manager.services[1].port
+        }
+        try:
+            params = urllib.parse.urlencode({'password': password})
```

Comment:
Move the first two instructions outside of the try-except block, since they can't raise HTTPError.